packages/Php73/tests/Rector/FuncCall/StringifyStrNeedlesRector/Fixture/skip_already_string.php.inc

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Rector\Php73\Tests\Rector\FuncCall\StringifyStrNeedlesRector\Fixture;
+
+final class SkipAlreadyString
+{
+    public function getNeedle(): string
+    {
+        return 'needle';
+    }
+}
+
+strpos('needle', (new SkipAlreadyString)->getNeedle());
```

Comment:
This better be wrapped in a method in the class, since fixtures are included (run), e.g.

```php
public function run(): string
{
    strpos('needle', (new self())->getNeedle());
}
```